SimpleGame/src/main/scala-2.11/edu/gemini/experimental/jluhrs/ConsoleKeys.scala

```diff
@@ -0,0 +1,41 @@
+package edu.gemini.experimental.jluhrs
+
+import edu.gemini.experimental.jluhrs.EventStream._
+import edu.gemini.experimental.jluhrs.Game._
+
+import scalaz.concurrent.Task
+import scalaz.stream.{Process, async}
+import scala.concurrent.duration._
+import sys.process._
+
+/**
+  * Created by jluhrs on 6/15/16.
+  */
+object ConsoleKeys {
+  // Build a process that produces player inputs from keystrokes
+  private val inputQ = async.boundedQueue[Event[PlayerInput]](10)
+
+  val inputP: Process[Task, Event[PlayerInput]] = inputQ.dequeue
```

Comment:
Note that this will let you have a single listener on the queue, that maybe fine depending on your use case. For the seqexec I used a topic instead since we have many clients